src/Common/src/TypeSystem/IL/Stubs/ILEmitter.cs

```diff
@@ -201,6 +204,189 @@ public void EmitSwitch(ILCodeLabel[] labels)
             }
         }
 
+        public void EmitLdInd(TypeDesc type)
+        {
+            switch (type.Category)
```

Comment:
OK. I'll add a special case for `TypeFlags.Enum` and recursive on `UnderlyingType` instead.